### DISCUSSION TITLE
how using ReportViewer to Uno

### DISCUSSION THREAD

**Discussion by AbdalaMask** [unoplatform/uno#7368] [Q&A]
### Current behavior

how using ReportViewer to Uno

**Comment by jeromelaban** [comment]
Thanks for the report. Can you explain what you mean by ReportViewer, specifically?

**Reply by AbdalaMask** [reply]
ReportViewer, file 
![image](https://user-images.githubusercontent.com/37013638/138346646-6b8ddc8a-966b-4eb0-9251-b30008731f2b.png)

**Reply by AbdalaMask** [reply]
![image](https://user-images.githubusercontent.com/37013638/138347099-74dd9e9a-7255-408c-862f-256f4ad50faf.png)

**Comment by AbdalaMask** [comment]
https://github.com/jogibear9988/CodeReason.Reports

**Comment by AbdalaMask** [comment]
how using BoldReports to uno
https://www.boldreports.com/
